index.html

```diff
@@ -289,7 +289,114 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+       <div id="contact">
+        <h2>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+        <form  id ="form" action="https://formspree.io/f/mwkakrqj" method="GET">
+          <input
+            type="text"
+            name="fullname"
+            id="fullname"
+            placeholder="Full name"
+            required
+          />
+          <div class="name-form">
+            <input
+              type="text"
+              name="name"
+              id="name"
+              placeholder="First name"
+              required
+            />
```

Comment:
Good job for adding the `required` attribute, so make sure to validate the name fields are not more than 30 characters.